src/engraving/dom/measurenumberbase.cpp

```diff
@@ -60,12 +59,7 @@ MeasureNumberBase::MeasureNumberBase(const MeasureNumberBase& other)
 
 engraving::PropertyValue MeasureNumberBase::getProperty(Pid id) const
 {
-    switch (id) {
-    case Pid::HPLACEMENT:
-        return int(hPlacement());
-    default:
-        return TextBase::getProperty(id);
-    }
+    return TextBase::getProperty(id);
```

Comment:
An override that only calls the parent class's method can/should be removed (unless you're expecting to put new stuff here in the very near future)